### PR TITLE
 new build w gnuradio3.9 [WIP]

### PR DESCRIPTION
**Pull request by petrushy**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by petrushy** [comment]
@conda-forge-admin, please rerender

**Comment by petrushy** [comment]
This build is still using gnuradio 3.8, not sure why it is not picking v3.9

**Comment by ryanvolz** [comment]
I was planning on holding off on this migration until a few more OOTs released updates to be compatible with 3.9. The way to do it is to change the conda-forge global pin, for which there is already a pull request to the pinning feedstock. Then everything will automatically get a migration PR. That said, if you want it now for just soapy, I'm not opposed. You'll need to add a pin to 3.9 for gnuradio_core to conda_build_config.yaml.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ requirements:
     - pkg-config  # [not win]
   host:
     - boost-cpp
-    - gnuradio-core
+    - gnuradio-core {{ gnuradio-core }}
```
Comment (ryanvolz) [inline recipe/meta.yaml]: This shouldn't be needed.

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+gnuradio-core:
```
Comment (ryanvolz) [inline recipe/conda_build_config.yaml]: Needs to use an underscore (for some YAML reason): `gnuradio_core`. Then you should see that it worked after a rerender.

**Review by ryanvolz — CHANGES_REQUESTED** [review]

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (petrushy) [reply recipe/conda_build_config.yaml]: thanks :)

**Comment by ryanvolz** [comment]
@conda-forge-admin, please rerender

**Comment by petrushy** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**